### PR TITLE
WithRatelimiterOptions() + redefine operation.Params.Timeout and operation.Params.CancelAfter in ratelimiter client

### PR DESCRIPTION
**Pull request by asmyasnikov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 * Fixed closing proxy clients
 * Added `ydb.Connection.With(opts ...ydb.CustomOption)` for taking proxy `ydb.Connection` with some redefined options
 * Added `ydb.MetaRequestType` and `ydb.MetaTraceID` aliases to internal `meta` package constants
+* Added `ydb.WithRatelimiterOperationTimeout()` and `ydb.WithRatelimiterOperationTimeout()` options
```
Comment (candiduslynx) [inline CHANGELOG.md:5]: * Added `ydb.WithRatelimiterOperationTimeout()` and `ydb. WithOperationCancelAfter()` options

File: options.go [hunk 2]
```diff
@@ -385,6 +385,20 @@ func WithTraceRatelimiter(trace trace.Ratelimiter) Option {
 	}
 }
 
+func WithRatelimiterOperationTimeout(timeout time.Duration) Option {
```
Comment (candiduslynx) [inline options.go:388]: `WithRatelimiterOptions(optss...rlconfig.Option)`

**Review by candiduslynx — CHANGES_REQUESTED** [review]

File: connection.go [hunk 3]
```diff
@@ -262,7 +262,20 @@ func New(ctx context.Context, opts ...Option) (_ Connection, err error) {
 
 	c.coordination = lazy.Coordination(c.db, c.coordinationOptions)
 
-	c.ratelimiter = lazy.Ratelimiter(c.db, c.ratelimiterOptions)
+	c.ratelimiter = lazy.Ratelimiter(
+		c.db,
+		append(
```
[the remark this answers is not in the document]
Comment (asmyasnikov) [reply connection.go:267]: Your code will replace custom ratelimiterOptions

File: options.go (hunk 2 above)
Comment (asmyasnikov) [reply options.go]: ack

File: connection.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (asmyasnikov) [reply connection.go:267]: discard

File: CHANGELOG.md (hunk 1 above)
Comment (asmyasnikov) [reply CHANGELOG.md]: ack

**Review by candiduslynx — DISMISSED** [review]

**Review by candiduslynx — APPROVED** [review]

**Pull request merged**